src/components/seekbar/seekbar.tsx

```diff
@@ -576,7 +580,7 @@ class SeekBar extends Component<any, any> {
         aria-valuemin={0}
         aria-valuemax={Math.round(this.props.duration)}
         aria-valuenow={Math.round(this.props.currentTime)}
-        aria-valuetext={`${toHHMMSS(this.props.currentTime)} of ${toHHMMSS(this.props.duration)}`}
+        aria-valuetext={`${getTimeInText(props.currentTime, props.player.config.ui, true)} of ${getTimeInText(props.duration, props.player.config.ui, true)}`}
```

Comment:
if you're only using locale from ui, pass just the locale, not the entire object. it's more readable.

Reply:
done